testsuite/MDAnalysisTests/analysis/test_psa.py

```diff
@@ -93,6 +93,16 @@ def test_dendrogram_produced(self):
         err_msg = "Dendrogram dictionary object was not produced"
         assert_(type(self.plot_data[1]) is dict, err_msg)
 
+    def test_dist_mat_to_vec_i_less_j(self):
+        """Test that distance is correct if i < j"""
```

Comment:
The comment is misleading: `dist_mat_to_vec` does not return a distance, but an index. The docstring of the function says:

> Convert distance matrix indices (in the upper triangle) to the index of
    the corresponding distance vector.